source/languages/en/riakcs/cookbooks/configuration/Configuring-an-S3-Client.md

```diff
@@ -12,6 +12,12 @@ This tutorial will show you how to use [s3cmd](http://s3tools.org/s3cmd)
 as an S3 client. While it won't cover all of the client's features, it
 will show you how to create a configuration and run some basic commands.
 
+>**Warning: s3cmd Signature Version**
+> If you are using s3cmd version 1.5.0 or greater you will need to append the
```

Comment:
In order for there to be a space in between you need an extra > between the Warning heading and the body text.